src/components/NcAppNavigationCaption/NcAppNavigationCaption.vue

```diff
@@ -136,7 +136,7 @@ export default {
 
 	&__title {
 		font-weight: bold;
-		color: var(--color-primary-element);
+		color: var(--color-main-text);
```

Comment:
That could be checked after https://github.com/nextcloud/server/pull/38159?notification_referrer_id=NT_kwDOAonkBbM2NDA2OTkwNjc3OjQyNTkxMjM3 again